### ISSUE TITLE
Photo

### ISSUE THREAD

**Issue by Hadson69** [TechnicalHeadquarter/saycheese#2]
Where can I see photos of victim?

**Comment by Bin-ahmed** [comment]
It doesn't work

**Comment by Joh2019** [comment]
No cam file recieved

**Comment by TechnicalHeadquarter** [comment]
> It doesn't work

I dont think it does not work, you should explain error and i will tell you solution.

**Comment by TechnicalHeadquarter** [comment]
> No cam file received

It is important that the victim must allow camera permission so you will get pics...

**Comment by Sayatidaktahu** [comment]
> > Tidak ada file kamera yang diterima
> 
> Penting bahwa korban harus mengizinkan izin kamera sehingga Anda akan mendapatkan foto ...

please provide to take video
